apps/aecontract/src/aect_contracts.erl

```diff
@@ -138,6 +139,17 @@ serialize(#contract{owner = OwnerId, referers = RefererIds} = C) ->
       , {deposit, deposit(C)}
       ]).
 
+-spec serialize_for_client(contract()) -> map().
+serialize_for_client(#contract{} = C) ->
+    #{ <<"id">> => aec_base58c:encode(contract_pubkey, id(C))
+     , <<"owner">> => aec_base58c:encode(account_pubkey, owner(C))
+     , <<"vm_version">> => vm_version(C)
+     , <<"log">> => log(C)
```

Comment:
This can be any arbitrary binary though for now it shall always be `<<>>` I believe (cc @happi ) so I suggest you keep it as it is for the moment - no encoding needed in this PR.